charts/node/values.yaml

```diff
@@ -449,6 +449,10 @@ node:
     #   mountPath: /etc/config/
     #   readOnly: true
 
+  # Enable Offchain Indexing.
+  # https://docs.substrate.io/fundamentals/offchain-operations/
+  offchainIndexing: false
```

Comment:
Can you please rename this to `enableOffchainIndexing`  to be consistent with the flag.

Reply:
Done. Thanks for reviewing.